### PR TITLE
Should fix #3

### LINKED ISSUE #3: PHP 5.4?
The bundle's composer.json says that the code only works in PHP 5.5+. Is this really the case that it cannot be ran on 5.4 for some objective reason? Not much changed between these versions, if I am not wrong.

It would be good if you changed composer.json to support PHP 5.4 (if that wont break anything, of course).

### PR DESCRIPTION
**Pull request by kachkaev**

### CODE REVIEW

**Comment by kachkaev** [comment]
#3

**Comment by kachkaev** [comment]
Looks like I missed a few more occurrences of ::class

File: DependencyInjection/CompilerPass/RouterCompilerPass.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ public function process(ContainerBuilder $container)
 
             //RouteCollection
             $collectionServiceName = 'gos_pubsub_router.collection.' . $name;
-            $collectionDef = new Definition(RouteCollection::CLASS);
+            $collectionDef = new Definition(get_class(RouteCollection));
```
Comment (jjsaunier) [inline DependencyInjection/CompilerPass/RouterCompilerPass.php:38]: `Gos\Bundle\PubSubRouterBundle\Router\RouteCollection`

File: DependencyInjection/CompilerPass/RouterCompilerPass.php [hunk 2]
```diff
@@ -72,7 +72,7 @@ public function process(ContainerBuilder $container)
             //Router context
             $contextConf = $routerConf['context'];
             $routerContextServiceName = 'gos_pubsub_router.context.' . $name;
-            $routerContextDef = new Definition(RouterContext::CLASS);
+            $routerContextDef = new Definition(get_class(RouterContext));
```
Comment (jjsaunier) [inline DependencyInjection/CompilerPass/RouterCompilerPass.php:75]: `Gos\Bundle\PubSubRouterBundle\Router\RouterContext`

**Pull request closed without merging**